lib/dirac_twisted_clover.cpp

```diff
@@ -296,7 +306,7 @@ namespace quda {
     }
 
     // we desire solution to full system
-    auto tmp = getFieldTmp(b[0].Even());
+    auto tmp = getFieldTmp(x[0].Even());
```

Comment:
Can the prepare/construct be multi-rhs-ified, or does the `create_alias` logic cause issues I haven't thought of?

Reply:
See here: https://github.com/lattice/quda/pull/1465#discussion_r1612742057